tests/testData/mockData.ts

```diff
@@ -68,6 +68,10 @@ interface GetMockRequestAndResponse {
     url?: any;
 }
 
+export const defaultSessionAttributes = {
+    [FARE_TYPE_ATTRIBUTE]: { fareType: 'single' },
+};
```

Comment:
If we attached these `defaultSessionAttributes` to the `getMockRequestAndResponse` method, we shouldn't need to pass `defaultSessionAttributes` into every test.